.github/workflows/ci.yaml

```diff
@@ -18,7 +18,7 @@ jobs:
         with:
           python-version: "3.9"
       - run: pip install cram nextstrain-augur
-      - run: cram --shell=/bin/bash tests/sanitize-metadata.t
+      - run: cram --shell=/bin/bash tests/sanitize-metadata.t tests/annotate_metadata_with_index.t
```

Comment:
Eventually, we want to run all Cram tests in the tests directory with a single `cram tests/` command, but the complexity and run-time of `different-inputs.t` prevents us from running all tests in each CI run. Resolving that issue is for a different PR.

Reply:
Today I learned there are Cram tests in ncov, too!